### PR TITLE
feat(teams): New team select page

### PR DESCRIPTION
**Pull request by moncapitaine**

### CODE REVIEW

File: app/backend/graphql/team/team.ts [hunk 1]
```diff
@@ -29,5 +30,10 @@ export const Team = objectType({
       description: 'List of all customers of the team',
       resolve: (team, _arguments, context) => context.prisma.customer.findMany({ where: { teamId: team.id } }),
     })
+    t.list.field('projects', {
+      type: Project,
+      description: 'List of all projects of the team',
+      resolve: (team, _arguments, context) => context.prisma.project.findMany({ where: { teamId: team.id } }),
```
Comment (PascalHelbig) [inline app/backend/graphql/team/team.ts:36]: @moncapitaine 
As a team member with the role "Member", am I allowed to see all the projects of the team?

Comment (moncapitaine) [reply app/backend/graphql/team/team.ts:36]: I understand your point but find it OK for now.

Comment (PascalHelbig) [reply app/backend/graphql/team/team.ts:36]: But then we should keep it consistent.
We have a similar query that behaves differently:
https://github.com/progwise/timebook/blob/main/app/backend/graphql/project/queries/projectsQueryField.ts

File: app/backend/graphql/team/queries/teamsQueryField.test.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      },
+    })
+  })
+
+  it('should return teams where the user is member of', async () => {
+    const testServer = getTestServer({ prisma, teamSlug: undefined })
+    const response = await testServer.executeOperation({ query: teamsQuery })
+    expect(response.data?.teams.length).toBe(2)
+    expect(response.data?.teams[0].id).toBe('team1')
+    expect(response.data?.teams[1].id).toBe('team2')
```
Comment (PascalHelbig) [inline app/backend/graphql/team/queries/teamsQueryField.test.ts:64]: Is there a reason why you only check certain fields from the response?

I would rather use `toEqual()` to test everything and not forget anything. But I know that these are personal preferences.

File: app/backend/graphql/team/team.ts [hunk 3]
```diff
@@ -29,5 +30,21 @@ export const Team = objectType({
...
+      type: Project,
+      description: 'List of all projects of the team',
+      resolve: (team, _arguments, context) =>
+        context.prisma.project.findMany({
+          where: {
+            AND: {
+              teamId: team.id,
+              OR: {
+                projectMemberships: { some: { userId: context.session?.user.id } },
+                team: { teamMemberships: { some: { role: 'ADMIN', userId: context.session?.user.id } } },
```
Comment (PascalHelbig) [inline app/backend/graphql/team/team.ts:43]: If the session is undefined, then these where clause will not be checked.
The case is rather unlikely, but if we can avoid it, maybe we should do it.

**Review by PascalHelbig — APPROVED** [review]
Just some smaller comments.

In general: I find it hard to review Tailwind Code. I think we should add screenshots in the future.
@moncapitaine: What do you think?

File: app/backend/graphql/team/team.ts (hunk 3 above)
Comment (moncapitaine) [reply app/backend/graphql/team/team.ts:47]: done

**Pull request merged**